scripts/system/libraries/networkingConstants.js

```diff
@@ -0,0 +1,14 @@
+//  networkingConstants.js
+//
+//  Created by Kalila L. on 8/27/20
+//  Copyright 2020 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+// Interface Metadata Source
+var interfaceMetadataSource = "https://cdn.vircadia.com/dist/launcher/vircadiaMeta.json";
```

Comment:
Is the metadata related only to Interface, or does it also relate to the Sandbox server?

Reply:
I've only ever considered it related to the interface but technically the domain-server gets bumped when the interface does so we can in fact update the domain-server to respond to this as well.

Reply:
Add an issue as a reminder to do this?